vendor/k8s.io/kubernetes/pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -362,6 +362,11 @@ func (gce *GCECloud) ScrubDNS(nameservers, searches []string) (nsOut, srchOut []
 	return nameservers, srchOut
 }
 
+// HasClusterID returns true if the cluster has a clusterID
+func (gce *GCECloud) HasClusterID() bool {
+	return true
```

Comment:
@rrati isn't GCE cloud using Initialize() to get the cluster id and this should return true only when the clusterID is really available?

Reply:
@mfojtik Yes, this really should be checking if the GCE Cluster ID is set or not.  I'll log an issue upstream and correct it.

Reply:
@mfojtik Actually, looking at the usage of GCE ClusterID this function should be as is, ie always returning true.